engine-appliance/scripts/get-boot-iso-url.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python3
+
+import os
+import yaml
+import argparse
+
+def main():
+    parser = argparse.ArgumentParser(prog="renderks")
```

Comment:
I guess 'renderks' might be a bug, no idea. At least clarifies where I copied this from :-). It probably makes sense to somehow unite them back to a single program, but I didn't bother.